app/feeds/new/license/template.hbs

```diff
@@ -7,91 +7,94 @@
 		Thanks for helping us catalog the contents of the feed. Now, please help us find any licenses or terms attached to the feed.
 	</p>
 </div>
-<div class="container input-form">
-	<div class="row">
-		<div class="col-sm-12 input-section">
-			<div class="input-form-label">Is a license attached to the feed?
-				<div class="btn-group" data-toggle="buttons">
-					{{license-form-toggle-button attr=license_present_mode text="Yes" value=true}} {{license-form-toggle-button attr=license_present_mode text="No" value=false}}
-				</div>
-			</div>
-		</div>
-	</div>
-	{{#if license_present_mode}}
+<div class="container">
+	<div class="input-form-wrapper">
```

Comment:
a lot of template changes are from renaming class, and making more use of bootstrap grid system.